config/config.go

```diff
@@ -41,3 +43,44 @@ func ParseConfig(tomlCfgFile string) (*Config, error) {
 	}
 	return &cfg, nil
 }
+
+// Option is a option setter function type used to pass varios configurations
+// to the controller
+type Option func(c *Config)
+
+// OptionDefaultNetwork function returns an option setter for a default network
+func OptionDefaultNetwork(dn string) Option {
+	return func(c *Config) {
+		c.Daemon.DefaultNetwork = dn
```

Comment:
Why don't you call `strings.TrimSpace` here and assign that value in the config instead of calling `strings.Trimspace` everywhere?